templates/lnurldevice/index.html

```diff
@@ -757,7 +757,7 @@ <h6 class="text-subtitle1 q-my-none">
       LNbits.api
         .request('GET', '/api/v1/currencies')
         .then(response => {
-          this.currency = ['USD', ...response.data]
+          this.currency = ['sat', 'USD', ...response.data]
```

Comment:
this is the only change a made, the rest is just running isort... accidently commited it to this PR